src/main/java/org/apache/sling/servlets/post/impl/operations/ModifyOperation.java

```diff
@@ -134,13 +163,13 @@ protected String getResourcePath(SlingHttpServletRequest request) {
             // check whether it is a create request (trailing /)
             if (suffix.endsWith(SlingPostConstants.DEFAULT_CREATE_SUFFIX)) {
                 suffix = suffix.substring(0, suffix.length()
-                    - SlingPostConstants.DEFAULT_CREATE_SUFFIX.length());
+                        - SlingPostConstants.DEFAULT_CREATE_SUFFIX.length());
```

Comment:
From here on there are a lot of formatting changes; it would be good to leave them out of the PR